EjercicioDiaUno.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjerciciosProyect_Visual.Net
+{
+    class EjercicioDiaUno
+    {
+        EjercicioDiaUno ejercicioDia;
+        public static void Main(string[] args)
+        {
+            EjercicioDiaUno ejercicioDia = new EjercicioDiaUno();
+
+            ejercicioDia.llamarEjercicioUno();
+            ejercicioDia.llamarEjercicioDos();
+            ejercicioDia.llamarEjercicioTres();
+        }
+
+        public void llamarEjercicioUno()
+        {
+            string resultado;
+
+            Console.WriteLine("Digite el Primer Número: ");
+            int numero1 = int.Parse(Console.ReadLine());
+            Console.WriteLine("Digite el Segundo Número: ");
+            int numero2 = int.Parse(Console.ReadLine());
+            Console.WriteLine("Digite el Tercer Número: ");
+            int numero3 = int.Parse(Console.ReadLine());
+
+            resultado = ejercicioDia.CalcularResultado(numero1, numero2);
+            Console.WriteLine("La Suma es: " + resultado);
+
+            resultado = ejercicioDia.CalcularResultado(numero1, numero2, numero3);
+            Console.WriteLine("La Multiplicación es: " + resultado);
+
+            Console.ReadKey();
+        }
+
+        public void llamarEjercicioDos()
+        {
+            Console.WriteLine("Digite un Número: ");
+            int cifra = int.Parse(Console.ReadLine());
+
+            cifra = ejercicioDia.CalcularUltimoDigito(cifra);
+
+            Console.WriteLine("El Número digitado es Negativo" + "; " + "Su ültimo digito es: " + (cifra % 10));
```

Comment:
Ya no es necesario realizar el cifra %10, ya que en la linea 46 estas llamando el CalcularUltimoDigito y estas actualizando la misma variable cifra.

```suggestion
            Console.WriteLine("El Número digitado es Negativo" + "; " + "Su ültimo digito es: " + cifra);
```

Reply:
Hecho y entendido